api/envoy/config/cluster/v3/outlier_detection.proto

```diff
@@ -18,7 +18,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // See the :ref:`architecture overview <arch_overview_outlier_detection>` for
 // more information on outlier detection.
-// [#next-free-field: 21]
+// [#next-free-field: 22]
```

Comment:
This needs a release note. Technically this is a breaking behavior change but I agree it's for the best, but we should call it out carefully in the release note.